api/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
```

Comment:
Did we move this here for any reason?

Reply:
Yes, it's the default config to be used, if you don't put one in your module's api test classpath. It doesn't affect webapp runtime.